developer-docs-site/package.json

```diff
@@ -23,6 +23,7 @@
     "@docusaurus/plugin-google-gtag": "2.1.0",
     "@docusaurus/preset-classic": "2.1.0",
     "@mdx-js/react": "1.6.22",
+    "aptos": "^1.3.13",
```

Comment:
I figure this was included accidentally?

Reply:
I did the `npm -i aptos` and this happened, predictably. I thought it is harmless to include it. Maybe I should remove it?